Purpose: Misc patches to manual pages
Authour: the Debian ALSA psychos
Fixes:   -
Status:  unknown

--- a/man/init.8
+++ b/man/init.8
@@ -39,21 +39,34 @@
 only a selected group of processes to exist.  The processes spawned by
 \fBinit\fP for each of these runlevels are defined in the
 \fB/etc/inittab\fP file.  \fBInit\fP can be in one of eight runlevels:
-\fB0\(en6\fP and \fBS\fP or \fBs\fP.  The runlevel is
+\fB0\(en6\fP and \fBS\fP (a.k.a. \fBs\fP).  The runlevel is
 changed by having a privileged user run \fBtelinit\fP, which sends
 appropriate signals to \fBinit\fP, telling it which runlevel to change
 to.
 .PP
-Runlevels \fB0\fP, \fB1\fP, and \fB6\fP are reserved. Runlevel 0 is used to
-halt the system, runlevel 6 is used to reboot the system, and runlevel
-1 is used to get the system down into single user mode. Runlevel \fBS\fP
-is not really meant to be used directly, but more for the scripts that are
-executed when entering runlevel 1. For more information on this,
+Runlevels \fBS\fP, \fB0\fP, \fB1\fP, and \fB6\fP are reserved.
+Runlevel S is used to initialize the system on boot.
+When starting runlevel S (on boot)
+or runlevel 1 (switching from a multi-user runlevel)
+the system is entering ``single-user mode'', after which the
+current runlevel is S.
+Runlevel 0 is used to halt the system;
+runlevel 6 is used to reboot the system.
+.PP
+After booting through S the system automatically enters one of
+the multi-user runlevels 2 through 5, unless there was some
+problem that needs to be fixed by the administrator in
+single-user mode.
+Normally after entering single-user mode
+the administrator performs maintenance and then reboots the system.
+.PP
+For more information,
 see the manpages for \fBshutdown\fP(8) and \fBinittab\fP(5).
 .PP
 Runlevels 7-9 are also valid, though not really documented. This is
 because "traditional" Unix variants don't use them.
-In case you're curious, runlevels \fIS\fP and \fIs\fP are in fact the same.
+.PP
+Runlevels \fIS\fP and \fIs\fP are the same.
 Internally they are aliases for the same runlevel.
 .\"}}}
 .PP
@@ -65,9 +78,10 @@
 entry (or no \fB/etc/inittab\fP at all), a runlevel must be
 entered at the system console.
 .PP
-Runlevel \fBS\fP or \fBs\fP bring the system to single user mode
-and do not require an \fB/etc/inittab\fP file.  In single user mode,
-\fB/sbin/sulogin\fP is invoked on \fB/dev/console\fP.
+Runlevel \fBS\fP or \fBs\fP initialize the system
+and do not require an \fB/etc/inittab\fP file.
+.PP
+In single user mode, \fB/sbin/sulogin\fP is invoked on \fB/dev/console\fP.
 .PP
 When entering single user mode, \fBinit\fP initializes the consoles
 \fBstty\fP settings to sane values. Clocal mode is set. Hardware
@@ -266,6 +280,14 @@
 for them.  If the processes change their group, \fBinit\fP can't
 kill them and you may end up with two processes reading from one
 terminal line.
+.PP
+On a Debian system, entering runlevel 1 causes all processes
+to be killed except for kernel threads and the script that does
+the killing and other processes in its session.
+As a consequence of this, it isn't safe to return from runlevel 1
+to a multi-user runlevel: daemons that were started in runlevel S
+and are needed for normal operation are no longer running.
+The system should be rebooted.
 .\"}}}
 .\"{{{  Diagnostics
 .SH DIAGNOSTICS
